app/assets/javascripts/angular/app.js

```diff
@@ -15,4 +15,6 @@ angular.module("Prometheus",
   $rootScope.alert = function(thing) {
     alert(thing);
   };
+}]).config(['$locationProvider', function($locationProvider) {
+   $locationProvider.html5Mode(true);
```

Comment:
bad news. this is required to make the custom pathing to work (`promdash.com/dashboard/profile`), but changing to html5 mode was shot down in this pr by @juliusv: https://github.com/prometheus/promdash/pull/422